### ISSUE TITLE
Resolving Conflicts

### ISSUE THREAD

**Issue by saharnazb** [jdebacker/ProblemSet9#24]
@adibalaram how did you solve conflicts? I cannot fetch the new updated scripts.

**Comment by adibalaram** [comment]
1. `git fetch upstream`
2. When you open your .py files in your editor, it should give you an option to keep your changes or accept what was done.
3. Then go through the usual process of adding files from your local branch.

Was that it @jtregde ?

**Comment by jtregde** [comment]
Yeah, if you attempt a merge and it tells you `CONFLICT` and something about the merge failing, then do a `git status -uno` and it will show you which file is causing the conflict. Then you can open that file and as @adibalaram said, your editor should show you where the changes are. After you make these changes, then you can add, commit and push as usual to resolve the conflict, then the merge should be able to work fine.

**Comment by saharnazb** [comment]
I did `git status` and made changes in firm but when came back to do rest I got this:

"fatal: not a git repository (or any of the parent directories): .git"
Trying to find out from google but if any idea, let me know

**Comment by jdebacker** [comment]
@saharnazb This usually means that you are not in a directory on your computer that is being tracked by git.

**Comment by saharnazb** [comment]
@jdebacker I checked the directory. Should I clone again?

**Comment by jdebacker** [comment]
@saharnazb Ideally, you don't want to have to do this.  Any idea where the directory with the .git file could have gone?

**Comment by saharnazb** [comment]
No. I even have all files and scripts in the same lacation.

**Comment by saharnazb** [comment]
Ok! I cloned and solved git repository problem but still have got the conflict but this time with several files. Thank you. I'll start over
